### Introduction to Python

In [3]:
3 * 8

24

In [4]:
(33+7)/5

8.0

The code 1+2 returns an integer and the code 1+2. returns a floating point number. The reason is that the decimal point in the second code is interpreted as a floating point number and the integer data type in Python cannot hold a floating point number.

In [5]:
for n in range(1,6):
    print(n**2)

1
4
9
16
25


In [7]:
i = 1
while i**2 < 1000:
    print(i**2)
    i += 1

1
4
9
16
25
36
49
64
81
100
121
144
169
196
225
256
289
324
361
400
441
484
529
576
625
676
729
784
841
900
961


At i=31 the value of i**2 is  
961
  and at i=32 the value of i**2 is  
1024
 . The value of i is  
32
  after the loop has completed.